client/control/tests/components/RecipeForm.js

```diff
@@ -28,4 +28,22 @@ describe('<RecipeForm>', () => {
       <RecipeForm params={{}} location={{ query: '' }} />
     </Provider>);
   });
+
+  describe('changeAction', () => {
+    const store = controlStore();
+    const recipeFormWrapper = mount(<Provider store={store}>
```

Comment:
These will be executed during test discovery instead of during the test itself; you should move them to a `beforeEach` function.